static/js/unread_ui.js

```diff
@@ -93,6 +93,13 @@ function consider_bankruptcy() {
                                            {unread_count: page_params.unread_msgs.count});
         $('#bankruptcy-unread-count').html(unread_info);
         $('#bankruptcy').modal('show');
+
+        if (window.electron_bridge === undefined) {
+            // normally this is handle in notifications.js but
+            // in this case we don't change the title so let the desktop
+            // app know the count.
+            window.electron_bridge.send_event('total_unread_count', page_params.unread_msgs.count);
+        }
```

Comment:
I don't think this is necessary -- if the user chooses the thing in the bankrupcty modal, we reload the page.

Reply:
yeah, I think this isn't actually necessary for that to work -- the other change should be sufficient.  

(After reading the symptoms, I think something else is going on in  zulip/zulip-electron#372).

Reply:
(There may be a secondary change we can make in about a week to fix zulip/zulip-electron#372, that depends on fetching all unread counts from the server)